test/fixtures/resource-discovery/widgets_cr.yml

```diff
@@ -0,0 +1,6 @@
+---
+apiVersion: "api.foobar.com/v1"
+kind: Widget
+metadata:
+  name: my-first-widget
+status: "ok"
```

Comment:
Why include status? Once it is a proper subresource (1.11+), this will be ignored.